frontend/src/components/Entries.js

```diff
@@ -1,21 +1,33 @@
 import PropTypes from 'prop-types'
 import React from 'react'
+import { useToggle } from 'react-hooks-lib'
+import { BsArrowBarDown, BsArrowBarUp } from 'react-icons/bs'
 import styled from 'styled-components/macro'
 import Divider from '../ui/Divider'
 
 export default function Entries({ entry }) {
+  const { on, toggle } = useToggle(false)
+
   return (
     <>
-      <ListStyled>
-        {entry.map((entry) => (
-          <label key={entry.id}>
-            <Divider />
-            <li>
-              <TopicStyled>{entry.topic}</TopicStyled>
-              <p>{entry.description}</p>
-            </li>
-          </label>
-        ))}
+      <ListStyled onClick={toggle}>
+        <label key={entry.id}>
+          <Divider />
+          <li>
+            {on || (
+              <CollapsedStyled>
+                <TopicStyled>{entry.topic}</TopicStyled>
+                <ArrowDownStyled />
+              </CollapsedStyled>
+            )}
+            {on && (
+              <CollapsedStyled>
+                <TopicStyled>{entry.topic}</TopicStyled> <BsArrowBarUp />
+                <p>{entry.description}</p>
+              </CollapsedStyled>
```

Comment:
evtl würde ich das gestylte element nicht collapsed styled nennen, da es ja nur in dem einen fall collapsed ist, aber das ist glaube ich auch geschmackssache...

Reply:
Danke, habe noch eine Möglichkeit gefunden, das rauszunehmen :)